### DISCUSSION TITLE
使用代码为 Xcode 工程添加文件 · /bei-xcodeproj-keng-de-zhe-ji-tian

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#490] [Announcements]
https://draveness.me/bei-xcodeproj-keng-de-zhe-ji-tian

**Comment by happiness9721** [comment]
補充一下
如果要加入resources(e.g. *.storyboard, *.xib, *.xcassets)的話，要使用這個

```
target.add_resources([file_ref])
```

**Comment by snote845** [comment]
请问下如果删除文件，执行这两行代码会出现找不到方法的问题可能是什么原因导致？
target.source_build_phase.remove_file_reference(file_ref)
target.headers_build_phase.remove_file_refernece(file_ref)

**Comment by huakucha** [comment]
headers_build_phase在add的时候怎么选择（project、private、public），而.h默认会加到project

**Comment by James-Chi-Jie** [comment]
不知道是不是我的网络有问题，这里的图显示不了

**Comment by leejunhui** [comment]
> @ArthurChi
> 不知道是不是我的网络有问题，这里的图显示不了

应该是图床挂了
